src/Billing/BillingProcessor/BillingClaimBatch.php

```diff
@@ -210,7 +215,8 @@ public function append_claim(&$segs)
             if ($elems[0] == 'GS') {
                 if ($this->bat_gscount == 0) {
                     ++$this->bat_gscount;
-                    $this->bat_content .= "GS*HC*" . $elems[2] . "*" . $elems[3] . "*$this->bat_yyyymmdd*$this->bat_hhmm*1*X*" . $elems[8] . "~";
+                    $this->bat_gs06 = $this->bat_icn + 1;
```

Comment:
ok, here is where I wonder if the padding will be kept

Reply:
you're right, the padding is lost but only the ISA needs it

Reply:
I'll make a better comment in the constructor and then note some of the other places where we're using it to increment the other segment counters